### PR TITLE
Fix jitteriness

### LINKED ISSUE #8501: Corpses shiver
forever

### PR DESCRIPTION
**Pull request by clusterfack**
Looks like it would only be removed from you if the life() tick removed it from you
Fixes #8501

### CODE REVIEW

File: code/modules/mob/mob.dm [hunk 1]
```diff
@@ -1772,5 +1772,10 @@ mob/proc/on_foot()
 /mob/proc/nuke_act() //Called when caught in a nuclear blast
 	return
 
+/mob/proc/remove_jitter()
+	if(jitteriness)
```
Comment (ghost) [inline code/modules/mob/mob.dm:1776]: why even have an if statement

Comment (clusterfack) [inline code/modules/mob/mob.dm]: Because I don't want to fuck up things being animated on a person for no reason

**Comment by Imagebard** [comment]
EVEN IN DEATH I (SHOULD STILL) JITTER.

**Comment by Exxion** [comment]
Does this also fix plasmamen never ever ever ceasing their jittering?

**Pull request merged**